src/sindri/config/website.py

```diff
@@ -139,19 +139,18 @@
     "load_current": [[0.1, 0.4, 0.6, 0.7, 1.5, 1.7, 1.8, 1.9],
                      STANDARD_COLORS[::-1] + STANDARD_COLORS[1:]],
     "temperature": [[-10, 0, 10, 40, 50, 60, 70], STANDARD_COLORS_TEMP],
-    "charge_state": [[i + 0.5 for i in range(8)],
-                     ["gray", "blue", "maroon", "orange", "red",
-                      "yellow", "lime", "green", "teal"]],
+    "charge_state": [[-0.5, 2.5, 3.5, 4.5, 8.5],
+                     ["red", "gray", "black", "red", "green", "red"]],
     "reference_voltage": [[12.5, 12.8, 13.1, 13.9, 14.9, 15.0, 15.1, 15.5],
                           STANDARD_COLORS[::-1] + STANDARD_COLORS[1:]],
     "load_state": [[i + 0.5 for i in range(5)],
                    ["gray", "green", "yellow", "orange", "red", "maroon"]],
     "alarm": [[-0.5, 0.5, 500], ["red", "green", "lime", "red"]],
-    "led_state": [[-0.5, 2.5, 6.5, 8.5],
-                  ["red", "gray", "green", "yellow", "red"]],
+    "led_state": [[-0.5, 2.5, 6.5, 9.5, 10.5, 11.5],
+                  ["red", "gray", "green", "yellow", "orange", "red", "maroon"]],
     "power_out": [[0.1, 6, 12, 24], STANDARD_COLORS[::-1]],
-    "power_load": [[1, 6, 9, 10, 15.5, 18, 19, 20, 24],
-                   STANDARD_COLORS[::-1] + ["teal"] + STANDARD_COLORS[1:]],
+    "power_load": [[1, 5, 11, 22, 26, 28],
+    			   ["black", "orange", "yellow", "gray", "green", "orange", "red"]], 
```

Comment:
```suggestion
    "power_load": [
        [1, 5, 11, 22, 26, 28],
        ["black", "orange", "yellow", "gray", "green", "orange", "red"],
        ], 
```

Fix indentation and erroneous mixed tabs and spaces